packages/admin-stories/src/admin-react-select/FinalFormReactSelectDialog.tsx

```diff
@@ -25,7 +25,6 @@ function Story() {
                                 <Field
                                     name="flavor"
                                     label="Flavor"
-                                    fieldContainerComponent={FieldContainerLabelAbove}
```

Comment:
In the current (old) version it is possible to implement a custom strategy how errors and labels are visualized. This is done by defining a fieldContainerComponent. This component can implement a "LabelAboveView" or "LabelOnTheSideView" or something else. This flexibility of the Field-Component is no longer supported with the changes here. The only choice now is "horizontal" or "vertical", and variant "vertical" is kind of deprecated anyway. Styles of course can be overwritten via the theme.

IMO this is a good change as it's easier to use now, and folks who need more flexibility can make their own Field-Component anyway. fieldContainerComponent-prop also was not really widely adopted by developers.